specification/resources/resource-manager/Microsoft.Features/stable/2021-07-01/examples/FeatureRegistration/SubscriptionFeatureRegistrationGET.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "providerNamespace": "subscriptionFeatureRegistrationGroupTestRG",
+    "featureName": "testFeature",
+    "api-version": "2020-09-01"
```

Comment:
@diwudd and @ifeoluwaokunoren - Please update the API version in all the example files to new api version 2021-07-01?

Reply:
Otherwise changes LGTM. I will signoff once you update the api version in all example files.